xep-0387.xml

```diff
@@ -244,6 +263,22 @@
           <td align='center'>&#10003;</td>
           <td>&xep0048;</td>
         </tr>
+        <tr>
+          <td><strong>Persistent Storage of Private Data via PubSub</strong></td>
+          <td align='center'>&#10005;</td>
+          <td align='center'>N/A</td>
+          <td align='center'>&#10003;&component;</td>
+          <td align='center'>N/A</td>
```

Comment:
The "advanced" client needs to be able to read and store PEP bookmarks as well. I'm not sure the regular PEP support from Core is sufficient for that, or if we should actually make a checkmark here.